source/retryable-writes/tests/README.rst

```diff
@@ -205,6 +205,9 @@ Each YAML file has the following keys:
       result object if their BulkWriteException (or equivalent) provides access
       to a write result object.
 
+      - ``errorLabelsContain``: A list of error label strings that the
+        error is expected to have.
```

Comment:
Per my other comments, I think we should also add `errorLabelsOmit` here if we introduce that syntax into the tests.